src/main/java/io/github/thebusybiscuit/slimefun4/utils/FireworkUtils.java

```diff
@@ -31,6 +33,8 @@ public final class FireworkUtils {
         Color.RED, Color.SILVER, Color.TEAL, Color.WHITE, Color.YELLOW
     };
     // @formatter:on
+    
+    private static final EntityType firework = VersionedEntityType.FIREWORK;
```

Comment:
Not sure why you make it a static variable when its only used once

Reply:
not sure either but can be cleaned up later